### ISSUE TITLE
Ed Sanders papers (C1703)

### ISSUE THREAD

**Issue by faithc** [pulibrary/aspace_helpers#310]
### User story
Alma records are needed for most of the containers associated with the Ed Sanders Papers (C1703), ideally by January 13, if that is possible. 

All of the data for the top containers is in ASpace _with the exception of_ boxes that need conservation work. I will submit another issue for these boxes when we get them back from P&C and they're ready to be sent to ReCAP. (We're planning to publish the finding aid/make the collection available before all of the materials are back from conservation since only a dozen or so boxes out of 400 are affected.) 

Please note there are gaps in the numbers; these gaps are in addition to the boxes that are undergoing conservation treatment.

### Basic Data
These are the things we'll need to run the import
- C1703 (mss repo) 
- Boxes 1-48, 50-52, 54-123, 125-147, 149-155, 157-160, 162-167, 169-177, 179, 181-193, 195-217, 219-238, 240-250, 252, 254-259, 300-313, 315-319, 321-336, 338-361, 363-367, 369-392, 394-413, 415-425, 427-428, 430-436, 438-440, 444-446, 449, 452, 457-459, 461-465. 
- date the top containers were last edited in ASpace (if known): seems like the records were last modified on 12/08/22.

### Additional Questions
- [No] does the collection include containers that already have item records in Alma?
- [n/a] if so, does the collection include duplicate box numbers?(If yes, please include barcodes to distinguish the ones we need to send.)
- [Yes] have you checked Alma to make sure the containers you're submitting don't have item records in Alma yet? (**Must be checked** before the import can be run.)

**Comment by regineheberlein** [comment]
@faithc Just confirming--all C1703 containers currently in ASpace should be included in this request, and none of them have item records in Alma?

**Comment by faithc** [comment]
@regineheberlein Sorry if the info I provided is unclear. No, the following containers that are currently in ASpace should _not be_ included in this request: 148, 180, 441, 443-445, 447, 448, 451, 453, 455, 456, 458-460. These are the boxes that P&C are currently treating. I'll submit another request for these once P&C is done with them. Want to be on the safe side in case anything needs to be rehoused. 

I checked Alma to be certain there are no item records. 

Please let me know if anything else is unclear.

**Comment by regineheberlein** [comment]
Ok, thanks! So since we are waiting for containers to come back from P&C, and they may potentially change, can it wait until all the containers are ready so I don't have to do this twice in short order?

I'm also noticing that the record is set to unpublished in ASpace--just so you're aware, that information doesn't come out with the export, so you may want to manually suppress the record in Alma once we've imported it.

**Comment by faithc** [comment]
The turnaround time for the boxes that are being treated may be somewhat significant and we were hoping to publish the finding aid and make the collection, with the exception of these few boxes, available around the end of the month. The other consideration is that it's going to likely take a couple of weeks to send the containers to ReCAP, hence the request to create the item records around the middle of the month. But if it's problematic to do this work in two batches I can check in with Alexis about waiting to publish the finding aid, though I'm not sure how long we'll be able to delay.

Also, thanks for the FYI about the need to suppress the Alma record once the item records have been added.

**Comment by alexisantracoli** [comment]
The finding aid needs to be published this month and should not wait for the completion of conservation treatment, which could be months.  I know it's not ideal, but, in this situation, it's really our only choice.  The other option I see is to describe the boxes in P&C now and just close them for treatment until they are available, but still they will need to be sent to ReCAP later.

**Comment by regineheberlein** [comment]
Sorry, can you explain how sending the containers to Alma (this ticket) relates to publishing the finding aid? I guess that's the point I'm missing?

**Comment by alexisantracoli** [comment]
Everything is going to ReCAP so they also need to go to Alma so they can be retrieved.

**Comment by regineheberlein** [comment]
I understand that part. I'm just not clear about how it ties into when you all open the collection. Sorry if that should be obvious.

**Comment by faithc** [comment]
No worries. Ideally, containers should be in the correct location when finding aids are published else there will be confusion if items are requested. 

Just to clarify, the materials in the boxes that are being treated are described; they'll just be restricted until P&C is done with treatment.

**Comment by alexisantracoli** [comment]
Got it.  The expectation has been that the collection would open in January to support a faculty member's teaching and research needs.   If the boxes in P&C were going to take a few weeks or even a month to deal with I'd delay it, but it could be many months and we can't delay it that long.

**Comment by regineheberlein** [comment]
Got it, thanks. So right now the ASpace records either don't identify the location or already say ReCAP?

**Comment by alexisantracoli** [comment]
Faith is the one with that information!

**Comment by faithc** [comment]
All of the containers have ReCAP assigned as their location; the ones that are being treated don't have barcodes assigned to them (yet).

**Comment by regineheberlein** [comment]
Ok, thanks for explaining that. So what I'm understanding in terms of the urgency of this ticket is this:
 
--the finding aid needs to open asap to address a faculty need
--some containers are currently in conservation treatment
--the collection will open before any conservation treatment is complete
--the container records not in conservation have been assigned the RecAP location
--therefore those containers need to be sent to that location before the collection opens

Does that capture it?

**Comment by faithc** [comment]
Yes, I think that covers everything.

**Comment by alexisantracoli** [comment]
Yes, that captures it.  We know there may be a bit of delay in getting it to ReCAP, but we're hoping to open it in February at the latest.

**Comment by regineheberlein** [comment]
Ok. I'll do my best to get to it by January 13, but since it requires manually sifting through the container records to exclude the ones that should be sent later, I can't promise that we'll have the time. In a pinch, could you update the top_container records with a temporary location of mss (or wherever the containers actually are right now) to facilitate requesting in the interim?

**Comment by regineheberlein** [comment]
Ah, thanks for explaining that it's at Mudd now but will be served out of Firestone. That really helps me understand where you all are coming from! We'll try to get it done.

**Comment by alexisantracoli** [comment]
No problem!!!  And, again, we're happy to do what we can to help out!

**Comment by faithc** [comment]
Thanks, Regine!

**Comment by regineheberlein** [comment]
Sudden priority justification: Collection is needed by researcher but is stored in a temp location from where it can't be serviced.

**Comment by regineheberlein** [comment]
@faithc 
Instead of searching for box numbers, can I assume that the 11 boxes currently in conservation are the same 11 boxes that don't have barcodes?

**Comment by faithc** [comment]
Yes you can. :)

**Comment by regineheberlein** [comment]
@faithc @alexisantracoli 
These 389 containers have been imported to the Alma Sandbox. Please QA and let me know if they are ready to imported to prod.

**Comment by faithc** [comment]
Amy, Phoebe, and I reviewed the records. The mostly look good to go, though we noticed the records don't have the call number associated with them. Would this data be added once they're in production? Thanks.

**Comment by regineheberlein** [comment]
Can you add a screenshot of where you expected additional data?

**Comment by regineheberlein** [comment]
@phoebeno 
How do these records compare to the ones for AC464?

**Comment by faithc** [comment]
![C1703_ItemRecords](https://user-images.githubusercontent.com/8058702/211372035-dc689ed6-4ff5-4e10-94f6-cf3de6714d5d.png)

I had checked another record, C1491, to confirm that call numbers were present: 
![C1491_ItemRecords](https://user-images.githubusercontent.com/8058702/211372218-b29a0d93-c702-40e4-9c28-a74efcea1a6e.png)

**Comment by regineheberlein** [comment]
These are coming from the physloc note, which the Ed Sanders ASpace record doesn't have. https://aspace.princeton.edu/staff/resources/4277#resource_notes_

**Comment by faithc** [comment]
Huh. Interesting that the call number comes from or is associated with the physloc note. I just added the note to the resource record. Thanks.

**Comment by phoebeno** [comment]
@regineheberlein I think the point is now moot, but the Griffin boxes did have the call number present in that column in Alma.

**Comment by regineheberlein** [comment]
@faithc @phoebeno @amycvo 
Re-imported and ready for another round of QA please.

**Comment by faithc** [comment]
Everything looks good to go. Thanks!

**Comment by regineheberlein** [comment]
This has been imported to prod.

**Comment by faithc** [comment]
Hi @regineheberlein - I'm not sure what happened in importing the item records to production, but @phoebeno noticed that only the 11 boxes that are with P&C, the boxes we're don't want added to Alma as of yet, are in Alma. The 389 boxes for which you created item records in the Alma sandbox are not in prod. Thanks for your help!

**Comment by faithc** [comment]
We had already started sending boxes to ReCAP so hopefully this doesn't cause any issues. eek.

**Comment by regineheberlein** [comment]
Odd--nothing changed since you QA'd it. I just reran the job, can you QA again please?

**Comment by faithc** [comment]
Thanks, Regine. The correct records are there now. Very odd.
